target/linux/generic/pending-6.6/981-gcc-plugins-stackleak-Avoid-.head.txt.section.patch

```diff
@@ -20,9 +20,9 @@ Signed-off-by: Kees Cook <keescook@chromium.org>
 
 --- a/scripts/gcc-plugins/stackleak_plugin.c
 +++ b/scripts/gcc-plugins/stackleak_plugin.c
-@@ -467,6 +467,8 @@ static bool stackleak_gate(void)
+@@ -469,6 +469,8 @@ static bool stackleak_gate(void)
  			return false;
- 		if (STRING_EQUAL(section, ".entry.text"))
+ 		if (STRING_EQUAL(section, ".head.text"))
```

Comment:
Please remove this patch it was added to v6.6.27 in https://github.com/gregkh/linux/commit/9dff96b8b3a404d3b3581af46450f2efeb7d290a

Reply:
Done!